src/YoutubeDownloader.php

```diff
@@ -252,16 +252,19 @@ public static function getDownloadUrlByFormats(array $avail_formats, $format)
 	 * @return array
 	 */
 	public static function createStreamMapFromVideoInfo(array $video_info)
-	{
-		if (isset($video_info['url_encoded_fmt_stream_map']) && isset($video_info['adaptive_fmts'])) {
-		  return [
-			explode(',', $video_info['url_encoded_fmt_stream_map']),
-			explode(',', $video_info['adaptive_fmts'])
-		  ];
-		}
+  {
```

Comment:
May I ask you to use proper indentation, preferably with tabs instead of spaces?